CHANGELOG.md

```diff
@@ -38,6 +58,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - UI: Replaced some Sass variables with CSS variables to allow customization of colors and fonts.
 - Public: Added new enterprise feature `logoCobrand`. When purchased and enabled allows integrator to provide their own logo image to be displayed alongside the Onfido logo.
 - Internal: Use Node 14 LTS for Travis to be consistent with `.nvmrc` and `Dockerfile`.
+- Internal: Enable `strict` mode in tsconfig.json
```

Comment:
This entry doesn't look like it belongs to this version

Reply:
It looks like we might've lost this CHANGELOG not between releases. I couldn't determine which version it was released in from the Jira ticket based on the PR. @ethan605 any idea which release this is meant to be under?

Reply:
It should be from `6.7.0`, because the PR was merged on 19 Mar, and `6.6.0` was released on 11 Mar :)

Reply:
Thanks! Updated the Github release notes for `6.7.0` now.